site/app/templates/grading/electronic/ta_status/StatusData.twig

```diff
@@ -61,20 +61,27 @@
             {% endif %}
                 <b>TA Grading By Sections:</b>
                 <div style="margin-left: 20px">
+                {# checks if any of sections has a verified value #}
                     {% for key, section in sections %}
                         {% if key != "peer_stu_grad"%}
                                 {% if include_bad_submissions  == 'include'  %}
-                                    {% if key == "NULL" and include_null_section != 'include' %}
-                                        {# Skip rendering this section #}
-                                     {% else %}
-                                        Section {{ key }}: {{ section.graded }} / {{ section.total }} ({{ section.percentage }}%)<br />
+                                    {% if not (key == "NULL" and include_null_section != 'include') %}
```

Comment:
It looks like this if-statement is duplicated in both branches of the outer conditional.  Should it be moved outside instead?

Reply:
Fixed in 5c53b4dbd69da2adb776128a462c64d482995bca